pypeit/par/pypeitpar.py

```diff
@@ -2554,7 +2542,7 @@ def valid_methods():
         Return the valid wavelength solution methods.
         """
         # TODO: Remove from this list anything that is not valid!
-        return ['simple', 'semi-brute', 'basic', 'holy-grail', 'identify', 'reidentify',
+        return ['simple', 'holy-grail', 'identify', 'reidentify',
```

Comment:
Yes! Be gone `basic` and `semi-brute`!  I don't think they are even used...

Reply:
They are not used... I did a thorough check of all non-deprecated code and found they are commented out.  I chose not to delete the commented code sections entirely, but maybe we can consider moving them to `deprecated/` somehow and remove the commented code from `pypeit/core/arc.py`.